SI/Sem.02/Pract.02/Task10.cpp

```diff
@@ -0,0 +1,52 @@
+﻿// Task10.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	double a, b;
+	char operation;
+	cout << "Enter the first number: ";
+	cin >> a;
+	cout << endl;
+
+	cout << "Choose possible operation from (+, -, *, /): ";
+	cin >> operation;
+	cout << endl;
+
+	cout << "Enter the second number: ";
+	cin >> b;
+	cout << endl;
+
+	double res = 0;
+	switch (operation)
+	{
+		case '+':
+			res = a + b;
+			break;
+		case '-':
+			res = a - b;
+			break;
+		case '*':
+			res = a * b;
+			break;
+		case '/':
+			if (b == 0)
+			{
+				cout << "The divisor cannot be 0" << endl;
+				break;
+			}
+			else
+			{
+				res = a / b;
+				break;
+			}
+		default:
+			cout << "Invalid operation" << endl;
+			break;
+	}
+	if (res != 0) cout << "Result: " << res << endl;
```

Comment:
This solution doesn't give output if your input is 5,+,-5 (the sum of 5 and -5 is zero)